docs/distro-compat.md

```diff
@@ -3,41 +3,125 @@
 ## Contents
 
 -   [Supported Linux Distros](#supported-linux-distros)
--   [Ubuntu Support](#ubuntu-support)
+-   [Sysbox Operational Modes](#Sysbox-Operational-Modes)
+-   [Kernel Upgrade Procedures](#Kernel-Upgrade-Procedures)
 
 ## Supported Linux Distros
 
-Sysbox relies on functionality that is currently only present in Ubuntu Linux.
-As a result Ubuntu is the only distro supported at this time. We are working
-on adding support for more distros.
+Sysbox relies on functionality that is only present in relatively modern Linux
+kernels.
 
-## Ubuntu Support
+In Ubuntu's case, these requirements are met starting with Kernel 5.0. For
+the rest of the Sysbox's supported distributions, Kernel 5.5+ is required.
 
-Sysbox requires recent Ubuntu versions:
+The following table summarizes the Linux distributions being supported as well
+as the operational modes (`shiftfs` vs `userns-remap`) being allowed. For
```

Comment:
In the sysbox [user-guide](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/security.md#user-namespace-id-mapping), we use "directed userns ID mapping" and "auto userns ID mapping". The former occurs when the container manager selects the ID mappings (e.g., docker with userns-remap, k8s with userns annotation, podman with UIDs enabled, etc). The latter is when sysbox selects the ID mapping (docker without userns-remap, podman without UIDs enabled). In the former sysbox does not require shiftfs (though it's OK if it's present in the kernel). In the latter sysbox requires shiftfs. 

I think this table should use the same terminology. In ubuntu, either mode works. In other distros, only directed ID mapping works.

Reply:
Agree, wip.